dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -10,19 +10,156 @@
+CWPKI2008E=CWPKI2008E: The ACME certificate authority directory URI must be a valid URI. The URI received was null or empty, received '{0}'.
+CWPKI2008E.explanation=Received a null or empty URI for the ACME certificate authority directory.
+CWPKI2008E.useraction=Enter a valid ACME certificate authority directory URI in the configuration.
+
+# "Error triggering challenge to ACME server."
+CWPKI2009E=CWPKI2009E: The challenge request to the ACME certificate authority at URI {0} failed. The error is {1}.
+CWPKI2009E.explanation=The challenge request failed and a certificate cannot be created.
+CWPKI2009E.useraction=Review the provided exception for details on the failure.
+
+# "Error updating challenge status from ACME server."
+CWPKI2010E=CWPKI2010E: The challenge update to the ACME certificate authority at URI {0} failed.The error is {1}.
+CWPKI2010E.explanation=The challenge update failed and a certificate cannot be updated.
+CWPKI2010E.useraction=Review the provided exception for details on the failure.
+
+# "Error creating order for ACME server."
+CWPKI2011E=CWPKI2011E: Failed to create the certificate order for the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2011E.explanation=Creating the certificate order failed and a signed certificate cannot be created.
+CWPKI2011E.useraction=Review the provided exception for details on the failure.
+
+# "Error signing the certificate signing request."
+CWPKI2012E=CWPKI2012E: Failed to sign the certificate signing request for the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2012E.explanation=The certificate order was created, but signing the certificate signing request failed.
+CWPKI2012E.useraction=Review the provided exception for details on the failure.
+
+# "Error ordering the certificate.
+CWPKI2013E=CWPKI2013E: The certificate signing request was created and signed, but the order failed from the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2013E.explanation=The certificate order was created and signed, but ordering the certificate failed.
+CWPKI2013E.useraction=Review the provided exception for details on the failure.
+
+#"Error getting the encoded certificate signing request."
+CWPKI2014E=CWPKI2014E: The certificate signing request for the ACME certificate authority at URI {0} was created and signed, but encoding the request failed. The error is {1}.
+CWPKI2014E.explanation=Encoding the certificate signing request failed and a signed certificate cannot be created.
+CWPKI2014E.useraction=Review the provided exception for details on the failure.
+
+# "Error updating the order status from the ACME server."
+CWPKI2015E=CWPKI2015E: Received an exception while requesting a certificate order update from the ACME certificate authority at URI {0}. The error is {1}.
+CWPKI2015E.explanation=A certificate order was send to the certificate authority. While requesting an status update, an exception occurred.
```

Comment:
**Change**
A certificate order was send to the certificate authority. While requesting an status update, an exception occurred.
 
**To***
A certificate order was sent to the certificate authority. While requesting a status update, an exception occurred.

or
An exception occurred while requesting a status update on a certificate order.